client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -123,12 +121,6 @@ public static void setupCookieJar(OkHttpClient.Builder clientBuilder)
         clientBuilder.cookieJar(new JavaNetCookieJar(new CookieManager()));
     }
 
-    public static void disableHttp2(OkHttpClient.Builder clientBuilder)
-    {
-        // Disable HTTP/2 as it's not tested nor supported
-        clientBuilder.protocols(ImmutableList.of(Protocol.HTTP_1_1, Protocol.HTTP_1_1));
-    }
-
```

Comment:
Is there a befit really to expose http 2 externally?

Reply:
It was exposed previously, I disabled it when I've added ALPN because of the lack of coverage